blockchain/contracts/ProofOfPhysicalAddress.sol

```diff
@@ -246,14 +266,12 @@ contract ProofOfPhysicalAddress {
         bool found;
         uint ai;
         bool confirmed;
-        (found, ai, confirmed) = userAddressByConfirmationCode(msg.sender, keccak256(confirmationCodePlain));
+        bytes32 keccakIdentifier;
+        (found, ai, confirmed, keccakIdentifier) = userAddressByConfirmationCode(msg.sender, keccak256(confirmationCodePlain));
         require(found);
+        require(!confirmed);
 
-        if (confirmed) {
-            revert();
-        } else {
-            users[msg.sender].physicalAddresses[ai].confirmationBlock = block.number;
-            totalConfirmed += 1;
-        }
+        registry.setClaim(msg.sender, keccakIdentifier, PhysicalAddressClaim.encode(block.number));
+        // totalConfirmed += 1;
```

Comment:
I think `totalConfirmed` might actually be useful, even though we store confirmations outside of PoPA now

Reply:
but we have no control if somebody removes the confirmation.

Reply:
In such a case we also need to remove `totalAddresses` and `totalUsers`, but I think we should keep them not as a counter of how many addresses/users there are at the moment, but how often is our service used.

Reply:
I kept the `totalConfirmed += 1;` for statistic purposes.